torch/csrc/jit/passes/vulkan_rewrite.cpp

```diff
@@ -213,6 +214,11 @@ void vulkanRemoveMutation(script::Module& module) {
   RemoveTensorMutation(graph);
 }
 
+void vulkanRunCanonicalOptimizations(script::Module& module) {
+  auto graph = module.get_method("forward").graph();
```

Comment:
I think we supports other optimization methods for CPU. What about vulcan?

Reply:
Yes, we should do the same for vulkan.